@marcelsavegnago agora eu acho que entendi o erro do travis, é tipo um teste o algo assim que ele esta executando e acaba gerando um erro, mas nada tem a ver com a instalação do módulo. Só para constar.

`2022-08-15 20:21:46,780 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: ERROR

2022-08-15 20:21:46,786 8583 INFO openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: ======================================================================

2022-08-15 20:21:46,786 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: ERROR: setUpClass (odoo.addons.l10n_br_contract.tests.test_l10n_br_contract.TestL10nBrContract)

2022-08-15 20:21:46,787 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: Traceback (most recent call last):

2022-08-15 20:21:46,787 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `   File "/home/travis/odoo-12.0/odoo/tests/common.py", line 191, in _handleClassSetUp

2022-08-15 20:21:46,787 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `     setUpClass()

2022-08-15 20:21:46,787 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `   File "/home/travis/build/OCA/l10n-brazil/l10n_br_contract/tests/test_l10n_br_contract.py", line 31, in setUpClass

2022-08-15 20:21:46,787 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `     cls.contract_id.recurring_create_invoice()

2022-08-15 20:21:46,787 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `   File "/home/travis/build/OCA/l10n-brazil/l10n_br_contract/models/contract_contract.py", line 175, in recurring_create_invoice

2022-08-15 20:21:46,787 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `     invoices = self._recurring_create_invoice()

2022-08-15 20:21:46,787 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `   File "/home/travis/dependencies/contract/contract/models/contract.py", line 635, in _recurring_create_invoice

2022-08-15 20:21:46,787 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `     invoices_values = self._prepare_recurring_invoices_values(date_ref)

2022-08-15 20:21:46,787 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `   File "/home/travis/build/OCA/l10n-brazil/l10n_br_contract/models/contract_contract.py", line 116, in _prepare_recurring_invoices_values

2022-08-15 20:21:46,787 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `     super_inv_vals = super()._prepare_recurring_invoices_values(date_ref=date_ref)

2022-08-15 20:21:46,788 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `   File "/home/travis/dependencies/contract/contract/models/contract.py", line 604, in _prepare_recurring_invoices_values

2022-08-15 20:21:46,788 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: `     invoice_values=invoice_values,

2022-08-15 20:21:46,788 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: ` TypeError: _prepare_invoice_line() got an unexpected keyword argument 'invoice_values'

2022-08-15 20:21:46,788 8583 INFO openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: Ran 0 tests in 2.865s

2022-08-15 20:21:46,788 8583 ERROR openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract: FAILED

2022-08-15 20:21:46,788 8583 INFO openerp_test odoo.addons.l10n_br_contract.tests.test_l10n_br_contract:  (errors=1)

2022-08-15 20:21:46,788 8583 ERROR openerp_test odoo.modules.module: Module l10n_br_contract: 0 failures, 1 errors`

--- a/l10n_br_contract/models/contract_contract.py
+++ b/l10n_br_contract/models/contract_contract.py
@@ -11,6 +11,7 @@ class ContractContract(models.Model):
     currency_id = fields.Many2one(
         readonly=False,
     )
+    country_id = fields.Many2one(related="company_id.country_id", store=True)
 
     @api.model
     def _fiscal_operation_domain(self):

--- a/l10n_br_contract/models/contract_line.py
+++ b/l10n_br_contract/models/contract_line.py
@@ -13,6 +13,7 @@ class ContractLine(models.Model):
     company_id = fields.Many2one(
         related="contract_id.company_id",
     )
+    country_id = fields.Many2one(related="company_id.country_id", store=True)
 
     fiscal_tax_ids = fields.Many2many(
         comodel_name="l10n_br_fiscal.tax",

--- a/l10n_br_contract/models/contract_line.py
+++ b/l10n_br_contract/models/contract_line.py
@@ -48,11 +48,3 @@ class ContractLine(models.Model):
                 self.discount / 100
             )
         return values
-
-    @api.model
-    def create(self, values):
-        res = super().create(values)
-        if res.contract_id.fiscal_operation_id and not res.fiscal_operation_id:
-            res.fiscal_operation_id = res.contract_id.fiscal_operation_id
-            res._onchange_fiscal_operation_id()
-        return res

--- a/l10n_br_contract/views/contract_line.xml
+++ b/l10n_br_contract/views/contract_line.xml
@@ -1,17 +1,98 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!-- Copyright 2020 KMEE
+     Copyright 2021 - TODAY, Escodoo
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo>
     <record id="contract_line_form_view" model="ir.ui.view">
-        <field name="name">contract.line form view (in contract)</field>
+        <field
+            name="name"
+        >contract.contract.line form view (in l10n_br_contract)</field>
         <field name="model">contract.line</field>
         <field name="inherit_id" ref="contract.contract_line_form_view" />
         <field name="arch" type="xml">
              <xpath expr="//field[@name='discount']/.." position="after">
-                 <group name="fiscal">
-                     <field name="fiscal_operation_id" />
-                     <field name="fiscal_operation_line_id" />
+                 <group name="fiscal_fields" invisible="1">
+                     <field name="fiscal_operation_type" invisible="1" readonly="1" />
+                     <field name="fiscal_genre_code" invisible="1" />
+                     <field name="tax_framework" invisible="1" />
+                     <field name="tax_icms_or_issqn" invisible="1" />
+                </group>
+                 <group
+                    name="fiscal"
+                    attrs="{'invisible': ['|', ('display_type', '=', 'line_section'), '&amp;', ('display_type', '=', 'line_note'), ('note_invoicing_mode', '!=', 'custom')]}"
+                >
+                    <field
+                        name="fiscal_operation_id"
+                        options="{'no_create': True}"
+                        attrs="{'invisible': [('country_id', '!=', %(base.br)d)]}"
+                    />
+                    <field
+                        name="fiscal_operation_line_id"
+                        options="{'no_create': True}"
+                        attrs="{'invisible': [('country_id', '!=', %(base.br)d)]}"
+                    />
+                    <field
+                        name="cfop_id"
+                        options="{'no_create': True}"
+                        attrs="{'invisible': ['|', ('tax_icms_or_issqn', '=', 'issqn'), ('country_id', '!=', %(base.br)d)]}"
+                    />
+                    <field
+                        name="service_type_id"
+                        options="{'no_create': True}"
+                        attrs="{'invisible': ['|', ('tax_icms_or_issqn', '=', 'icms'), ('country_id', '!=', %(base.br)d)]}"
+                    />
+                    <field
+                        name="city_taxation_code_id"
+                        attrs="{'invisible': ['|', ('tax_icms_or_issqn', '=', 'icms'), ('country_id', '!=', %(base.br)d)]}"
+                        options="{'no_create': True}"
+                    />
+                    <field
+                        name="cnae_id"
+                        attrs="{'invisible': ['|', ('tax_icms_or_issqn', '=', 'icms'), ('country_id', '!=', %(base.br)d)]}"
+                        options="{'no_create': True}"
+                    />
                  </group>
+            </xpath>
+            <xpath expr="//sheet" position="inside">
+                <field name="country_id" invisible="1" />
+                <notebook
+                    attrs="{'invisible': ['|', ('display_type', '=', 'line_section'), '&amp;', ('display_type', '=', 'line_note'), ('note_invoicing_mode', '!=', 'custom')]}"
+                >
+                    <page
+                        name="fiscal_taxes"
+                        string="Taxes"
+                        attrs="{'invisible': [('country_id', '!=', %(base.br)d)]}"
+                    />
+                    <page
+                        name="amounts"
+                        string="Amounts"
+                        attrs="{'invisible': [('country_id', '!=', %(base.br)d)]}"
+                    >
+                        <group>
+                            <group>
+                                <field name="amount_untaxed" />
+                                <field name="amount_fiscal" />
+                                <field name="amount_tax" />
+                                <field name="estimate_tax" readonly="1" />
+                            </group>
+                            <group>
+                                <field name="amount_total" />
+                                <field name="amount_tax_withholding" readonly="1" />
+                                <field name="amount_taxed" />
+                            </group>
+                        </group>
+                    </page>
+                    <page
+                        name="fiscal_line_extra_info"
+                        string="Extra Info"
+                        attrs="{'invisible': [('country_id', '!=', %(base.br)d)]}"
+                    />
+                </notebook>
+            </xpath>
+            <xpath expr="//group[@name='recurrence_info']" position="attributes">
+                <attribute name="attrs">
+                    {'invisible': ['|','|', ('display_type', '=', 'line_section'), ('parent.line_recurrence', '!=', True), '&amp;', ('display_type', '=', 'line_note'), ('note_invoicing_mode', '!=', 'custom')]}
+                </attribute>
             </xpath>
         </field>
     </record>

--- a/l10n_br_contract/views/contract_view.xml
+++ b/l10n_br_contract/views/contract_view.xml
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 KMEE
+     Copyright 2021 - TODAY, Escodoo
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo>
     <record id="contract_contract_form_view" model="ir.ui.view">
         <field name="name">contract.contract form view (in l10n_br_contract)</field>
@@ -6,12 +9,29 @@
         <field name="inherit_id" ref="contract.contract_contract_form_view" />
         <field name="arch" type="xml">
             <field name="fiscal_position_id" position="after">
-                <field name="fiscal_operation_id" />
+                <field name="country_id" invisible="1" />
+                <field
+                    name="fiscal_operation_id"
+                    options="{'no_create': True}"
+                    attrs="{'invisible': [('country_id', '!=', %(base.br)d)]}"
+                />
             </field>
             <xpath expr="//field[@name='contract_line_ids']" position="attributes">
                 <attribute
                     name="context"
                 >{'default_fiscal_operation_id': fiscal_operation_id, 'default_partner_id': partner_id, 'default_company_id': company_id}</attribute>
+            </xpath>
+            <xpath
+                expr="//field[@name='contract_line_ids']/tree/button"
+                position="before"
+            >
+                <field name="country_id" invisible="1" />
+                <field
+                    name="fiscal_tax_ids"
+                    widget="many2many_tags"
+                    options="{'no_create': True}"
+                    attrs="{'invisible': [('country_id', '!=', %(base.br)d)]}"
+                />
             </xpath>
         </field>
     </record>

